templates/report/dashboard.html

```diff
@@ -96,12 +96,28 @@ <h1>Dashboard for {{ serverName }}</h1>
                 'deployment': deployment.id,
                 'service': service.id,
                 'report_type': report_type,
-                'csrf_token': $("#csrf_token").val()
+                'csrf_token': $("#csrf_token").val(),
+                'data_num': typeof loadData.dataNum == 'undefined'? -1 : loadData.dataNum
             },
             dataType: "json"
         }).done(function (data) {
             if (data.status === "success") {
                 ajax.html(data.html).show();
+                console.log('before')
+                console.log(loadData.dataNum)
+                if(typeof loadData.dataNum == 'undefined'){
```

Comment:
Please try to style consistently with the other code

```suggestion
                if(typeof loadData.dataNum == "undefined") {
```